tracpro/templates/workflow_tracker_steps.html

```diff
@@ -0,0 +1,40 @@
+<div class="row workflow-tracker">
+    <a href="{% url 'groups.region_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 1 %}circle-selected{% else %}circle{% endif %}">
+            1
+            <div class="circle-label">
+                Select Panels
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'groups.group_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 2 %}circle-selected{% else %}circle{% endif %}">
+            2
+            <div class="circle-label">
+                Select Cohorts
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'polls.poll_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 3 %}circle-selected{% else %}circle{% endif %}">
+            3
+            <div class="circle-label">
+                Select Flows
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'orgs_ext.org_fetchruns' %}">
```

Comment:
I believe that staff and super users should be able to get to any of these pages so that's why I didn't include it to begin with.